.travis.yml

```diff
@@ -8,13 +9,13 @@ sudo: false
 
 before_install:
   - go get golang.org/x/tools/cmd/vet
-  - go get github.com/golang/lint/golint
+  - go version | grep -qE '(go1.5|go1.6)' && go get github.com/golang/lint/golint
```

Comment:
yeah, but now i'm trying to figure out the proper subterminal return codes,
because this is now failing as grep is not finding the version

On Sat, Mar 19, 2016 at 11:50 AM, W. Trevor King <notifications@github.com>
wrote:

> In .travis.yml
> <https://github.com/opencontainers/specs/pull/352#discussion_r56750650>:
>
> > @@ -8,13 +9,13 @@ sudo: false
> >
> >  before_install:
> >    - go get golang.org/x/tools/cmd/vet
> > -  - go get github.com/golang/lint/golint
> > +  - go version | grep -qE '(go1.5|go1.6)' && go get github.com/golang/lint/golint
>
> grep -q 'go1.[56]' should be sufficient.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/opencontainers/specs/pull/352/files/60c7a94edb5c008ef08bbfecaf22f4758d5a242d#r56750650>
>